tensorflow/core/kernels/decode_raw_op.cc

```diff
@@ -87,14 +87,21 @@ class DecodeRawOp : public OpKernel {
     } else {
       // Otherwise, the data is not in the host's byte order, and rather than a
       // direct copy, we need to reverse the byte ordering of each element.
+      int64 element_size;
+      if (DataTypeString(out_type_) == "complex64" || DataTypeString(out_type_) == "complex128") {
```

Comment:
Can you change the test to `(out_type_ == DT_COMPLEX64) || (out_type_ == DT_COMPLEX128)` in order to avoid the string conversion?

Reply:
Corrected, Thank you!